### ISSUE TITLE
v1.4 chat interface

### ISSUE THREAD

**Issue by Joe-Klauza** [Joe-Klauza/sandstorm-admin-wrapper#48]
Since v1.4 is adding logging for in-game chat, we can use that to add a couple of features:

- Chat interface in the server control page
  - Show live chat, much like the RCON-only logs
  - User input can be sent to the in-game chat (via RCON say), possibly prefixed with the SAW username to differentiate multiple `ADMIN`s
- Admin commands via chat
  - Since the chat logs have a user's Steam ID attached, we could easily compare this against the server's `Admins.txt` to allow them to issue commands.
  - Limited RCON commands (access to a subset of RCON commands; likely a good idea since we probably don't want to "exit" if chat is working...)

**Comment by kriansa** [comment]
For admin commands specifically:

- Can we hide the commands from the other connected players?

**Comment by Joe-Klauza** [comment]
>Can we hide the commands from the other connected players?

@kriansa I don't think so. Since all we receive is what is logged via in-game chat, by the time it ends up in the logs the other players can already see it. This would have to be implemented by NWI somehow.

Any admins with access to sandstorm-admin-wrapper would be able to use the RCON input on the server control page for this (though I realize the convenience of the in-game interface is lost).

**Comment by Joe-Klauza** [comment]
Improved by [v0.10.0-beta](https://github.com/Joe-Klauza/sandstorm-admin-wrapper/releases/tag/v0.10.0-beta) (adds chat to server control).

No admin commands are available via chat yet.
